index.js

```diff
@@ -40,6 +39,7 @@ app.listen(PORT, HOST, () => {
   console.log(`Starting Proxy at ${HOST}:${PORT}`);
 });
 
-//test req:
-// curl localhost:3006/info
-//curl localhost:3006/localhost:3002/booking/1
\ No newline at end of file
+//test my proxy:
+//curl http://localhost:3002/booking?campId=0
```

Comment:
this is not going to test your proxy becuase its making a direct request to the service, whic. is at 3002. a proxy test should lead with http://localhost:2000